docs/reference/glossary.md

```diff
@@ -264,7 +272,11 @@ containers.
 
 ## Toolbox
 
-Docker Toolbox is the installer for Mac and Windows users.
+[Docker Toolbox](https://docs.docker.com/toolbox/overview/) is a legacy installer for Mac and Windows users. It uses Oracle VirtualBox for virtualization.
+
+For Macs running OS X El Capitan 10.11 and newer macOS releases, [Docker for Mac](https://docs.docker.com/docker-for-mac/) is the better solution.
+
+For Windows systems running Windows 10 Pro and Microsoft Hyper-V, [Docker for Windows](https://docs.docker.com/docker-for-windows/) is the better solution.
```

Comment:
For Windows 10 systems that support Hyper-V (Professional, Enterprise and Education)...